File: gpao-tex-core/production/src/main/java/com/gpro/consulting/production/dto/FicheColisageValue.java
package com.gpro.consulting.production.dto;

import java.util.Calendar;
import java.util.List;

public record FicheColisageValue(
        Long id,
        Long orderId,
        List<ColisValue> colis,
        List<DetailsColisageValue> detailsColisages,
        String observations,
        Long nombreColis,
        Long quantiteColis,
        String numeroOf,
        Long quantiteTotale,
        Long produitId,
        Long clientId,
        String clientAbreviation,
        String clientReference,
        String couleur,
        String produitDesignationInfo1,
        String numeroInterne,
        Calendar dateDe,
        Calendar dateA

) {
}

File: gpao-tex-ui/source/main/src/app/services/order/models/fiche-colisage-value.ts
/* tslint:disable */
/* eslint-disable */
import { ColisValue } from '../models/colis-value';
import { DetailsColisageValue } from '../models/details-colisage-value';
export interface FicheColisageValue {
  clientAbreviation?: string;
  clientId?: number;
  clientReference?: string;
  colis?: Array<ColisValue>;
  couleur?: string;
  dateA?: string;
  dateDe?: string;
  detailsColisages?: Array<DetailsColisageValue>;
  id?: number;
  nombreColis?: number;
  numeroInterne?: string;
  numeroOf?: string;
  observations?: string;
  orderId?: number;
  produitDesignationInfo1?: string;
  produitId?: number;
  quantiteColis?: number;
  quantiteTotale?: number;
}
